strawberry/field.py

```diff
@@ -103,19 +118,46 @@ class strawberry_field:
     allowing us to us both syntaxes.
     """
 
-    def __init__(self, *, is_subscription=False, **kwargs):
-        self.field = dataclasses.field()
+    def __init__(
+        self,
+        *,
+        is_input=False,
+        is_subscription=False,
+        resolver=None,
+        name=None,
+        description=None,
+        metadata=None,
+    ):
+        self.field_name = name
+        self.field_description = description
+        self.field_resolver = resolver
         self.is_subscription = is_subscription
-        self.description = kwargs.get("description", None)
-        self.name = kwargs.pop("name", None)
-        self.kwargs = kwargs
+        self.is_input = is_input
+
+        super().__init__(
+            # TODO:
```

Comment:
What is this TODO left here?

Reply:
I need to support default values when using field, forgot about it :D